lib/ui/painting.dart

```diff
@@ -2633,6 +2633,7 @@ class ColorFilter {
 
   // The type of SkColorFilter class to create for Skia.
   // These constants must be kept in sync with ColorFilterType in paint.cc.
+  // ignore:unused_field
   static const int _TypeNone = 0; // null
```

Comment:
Rather than adding an ignore, let's just remove the unused field.

We should also do the same in the web_ui painting.dart

Reply:
There is no _TypeNone in web_ui painting.dart as far as I can tell

Reply:
> Rather than adding an ignore, let's just remove the unused field.

Done

Reply:
It's in `lib/web_ui/lib/src/engine/color_filter.dart`, line 118

Reply:
Ah, got you. Interesting this is not picked up by analyzer as unused field.